ReactSkia/sdk/NotificationCenter.h

```diff
@@ -60,9 +62,11 @@ class NotificationCenter {
 
         NotificationCenter(const NotificationCenter&) = delete;  
         const NotificationCenter& operator = (const NotificationCenter&) = delete;
+        folly::ScopedEventBaseThread eventBaseThread_;
```

Comment:
can we have purpose of eventBase thread reflects in naming like : eventEmitterThread_ ...